README.rst

```diff
@@ -302,24 +328,67 @@ proposal is ready for decision.
 
    * post their recommendation, with a rationale, on the GitHub discussion thread,
    * label the pull request as ``Pending committee review``,
-   * notify the committee by mentioning ``@ghc-proposals/ghc-steering-committee``,
-   * include the sentence “This proposal will now be discussed by the committee.
-     We welcome all authors to join the discussion, but kindly ask others to
-     refrain from posting.” in the comment
+   * re-title the proposal, appending ``(under review)`` at the end. (This enables easy email filtering.)
```

Comment:
If the proposal ends up reverting to the discussion phase, `(under review)` should be removed again.